### PR TITLE
Minor typos/redundancies/grammar fixes on page 4

### PR DESCRIPTION
**Pull request by chriskorosu**

### CODE REVIEW

File: problemsets-mat223/modules/module1.tex [hunk 1]
```diff
@@ -337,9 +337,8 @@
 we consider it to point in no directions.  It depends on our mood---but we must
 never talk about \emph{the} direction of the zero vector, since it's not defined.
 
-Formalizing, for vectors $\vec u$, $\vec v$, $\vec w$ and scalars $\alpha$ and $\beta$
-are scalars, the
-following are laws are always satisfied:
+Formally, for vectors $\vec u$, $\vec v$, $\vec w$, and scalars $\alpha$ and $\beta$, the
```
Comment (siefkenj) [inline problemsets-mat223/modules/module1.tex]: `Formally` -> `Formalizing`

**Comment by siefkenj** [comment]
After you change `Formally` to `Formalizing`, I will merge your change :-).

**Comment by chriskorosu** [comment]
Done!

**Comment by siefkenj** [comment]
Thanks! You can add your name to the contributors list, if you like :-D.

**Pull request merged**